src/sass/main.scss

```diff
@@ -3,8 +3,8 @@
 @import './partials/buttons';
 @import './partials/header-nav';
 @import './partials/mobile-menu';
-@import './partials/places';
```

Comment:
As @KJachimstal said, these import need to be added at the end of the list. Rest is fine. Good job:)